package.json

```diff
@@ -22,7 +22,6 @@
       "packages/project",
       "packages/bundler",
       "packages/todomvc",
-      "packages/convergence",
```

Comment:
This package doesn't exist anymore?

Reply:
I don't think so no. We may me keeping it around so that we can eventually deprecate it with the release of 1.0? /cc @minkimcello In any event, let's address it separately.